src/components/EditorToolbar.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { Container, Row, Col, Button } from 'react-bootstrap';
+import { IoIosPhonePortrait } from '@react-icons/all-files/io/IoIosPhonePortrait';
+import { IoIosPhoneLandscape } from '@react-icons/all-files/io/IoIosPhoneLandscape';
+import { IoIosMoon } from '@react-icons/all-files/io/IoIosMoon';
+import { IoSunnyOutline } from '@react-icons/all-files/io5/IoSunnyOutline';
+import { BiExpand } from '@react-icons/all-files/bi/BiExpand';
+import { IoPlay } from '@react-icons/all-files/io5/IoPlay';
+import { CgUndo } from '@react-icons/all-files/cg/CgUndo';
+import { IoIosArrowDown } from '@react-icons/all-files/io/IoIosArrowDown';
+//Code Editor
+import AceEditor from 'react-ace';
+import "ace-builds/src-noconflict/theme-tomorrow";
+import "ace-builds/src-noconflict/theme-monokai";
+
+export default function EditorToolbar(){
+
+    const buttonTheme = {
+        colors: {
+            main: "#404040",
+            text: "#ffffff"
+        },
+        fonts: {
+            body: "Poppins, sans-serif"
+        },
+        fontWeights: {
+            body: 400,
+            bold: 700
+        },
+        fontSizes: [14, 16, 24, 32, 48, 64, 72],
+        width: [150, 200, 250, 300, 350, 400, 450, 500],
+        spacing: [25, 35, 45]
+    }
+
+    const buttonStyle = {
```

Comment:
I see that we are using the native react styling and using it directly on HTML Elements. Didn't we agree on either using individual SCSS files or using Styled components?

Could you look into moving this into Styled components?